test_db.py

```
@@ -0,0 +1,9 @@
+from sqlalchemy import create_engine
+print("Connecting...")
+try:
+    engine = create_engine('postgresql://postgres:postgres@localhost:5433/personal_guru')
+    conn = engine.connect()
+    print("Success")
+    conn.close()
+except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)